setup/general/kuksa.val/README.md

```diff
@@ -21,3 +21,6 @@ vss.json
 ```
 
 You can find out, where to get these files [here](https://github.com/GENIVI/iot-event-analytics/docker/kuksa.val2iotea/README.md)
+
+For building vss.json and setting required values you can use helper script (./build-vss.sh)
```

Comment:
1) 'build-vss.sh' --> is it generate-vss.sh?

2) https://github.com/GENIVI/iot-event-analytics/docker/kuksa.val2iotea/README.md does not work.
-> https://github.com/GENIVI/iot-event-analytics/blob/develop/docker/vss2iotea/README.md

or https://github.com/GENIVI/iot-event-analytics/blob/develop/docker/kuksa.val2iotea/README.md - how it will look like after merging the rename vss to kuksa branch

Reply:
Updated iot-event-analytics version, which contains name and folder changes